docs/experimental/functions.md

```diff
@@ -177,6 +180,54 @@ apis:
       db: redisDbSample
       cmd: redisCmdSample
 ```
+## sample_include_match_all_filter
+
+Similar to the supported smaple_include_filter but will create samples only when ALL the specified filter keys and values are present in the sample.
```

Comment:
Rewrite to:

Similar to the supported `sample_include_filter`. It creates samples only when **all** the specified filter keys and values are present in the sample.